src/ArrangerConduit.sol

```diff
@@ -0,0 +1,251 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { console2 as console } from "../lib/forge-std/src/console2.sol";
+
+import { IArrangerConduit } from "./interfaces/IArrangerConduit.sol";
+
+interface ERC20Like {
+    function balanceOf(address src) external view returns (uint256 wad);
+    function transfer(address dst, uint wad) external returns (bool);
+    function transferFrom(address src, address dst, uint wad) external returns (bool);
+}
+
+contract ArrangerConduit is IArrangerConduit {
+
+    // TODO: Add and test Router ACL
+
+    /**********************************************************************************************/
+    /*** Declarations and Constructor                                                           ***/
+    /**********************************************************************************************/
+
+    address public override admin;
+    address public override fundManager;
+
+    mapping(address => uint256) public override outstandingPrincipal;
+    mapping(address => uint256) public override startingFundRequestId;
+    mapping(address => uint256) public override totalInterestEarned;
+    mapping(address => uint256) public override totalPositions;
+    mapping(address => uint256) public override totalWithdrawable;
+
+    mapping(address => FundRequest[]) public fundRequests;
+
+    mapping(bytes32 => mapping(address => uint256)) public override maxWithdraw;
+    mapping(bytes32 => mapping(address => uint256)) public override pendingWithdrawals;
+    mapping(bytes32 => mapping(address => uint256)) public override positions;
+
+    constructor(address admin_, address fundManager_) {
+        admin       = admin_;
+        fundManager = fundManager_;
```

Comment:
We should be able to update the fund manager. In fact this should not be set in the constructor at all and set in a regular dss `file(...)` function.

Reply:
Still an issue. `arranger`, `registry` and `roles` should all be set via `file(...)` and none of these should be configured in the constructor. We usually set these up in the spell.

Reply:
Updated